github-profiles/kenichi-7356/index.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <link rel="stylesheet" type="text/css" href="main.css">
+    <title>Perfil profesional</title>
+</head>
+<body>
+    <div class="header">
+    <h1><strong>Hi There</strong></h1>
+    </div>
+    <hr>
+   <!-- <dl>
+        <a href="#about"><h3><dt>About me</dt></h3></a>
+        <a href="#week1"><h3><dt>Bootcamp first week</dt></h3></a>
+        <h3><dd>- Creating alias for use in terminal</dd></h3>
+        <h3><dd>- Most used command in terminal</dd></h3>
+        <h3><dd>- HTML</dd></h3>
+    </dl>
+    <hr>
+-->
+    <h3 id="about">About me:</h3>
```

Comment:
Try to indent this `h3` element.

Reply:
Thank you! Your comments are very important to me.